cfme/intelligence/reports/reports.py

```diff
@@ -176,20 +178,22 @@ def all(cls, browser):
 
     @property
     def is_displayed(self):
+        expected_tree = [
```

Comment:
Question, isn't this under the "Saved Reports" navigation tree

Reply:
In this case, no, its the `Saved Reports` tab on a `Report` details page.